source/Camera/InfoUtilOther/ZividBenchmark/ZividBenchmark.cpp

```diff
@@ -226,10 +228,13 @@ namespace
 
     size_t getMinExposureTime(const std::string &modelName)
     {
+        size_t exposureTime;
```

Comment:
Same here. Looking at those two codes (before and after) it is clearly better before.

You should not change to something worse just because a tool tells you to.

In that case you need to shout and scream, ether you find a way to improve the code and if no one can tell you a better way or explain why we want this warning, then the diagnostics should be disabled.

Reply:
I didn't like it, but thought maybe you wanted the else..
I almost proposed the following:
```cpp
        size_t exposureTime = exposureTime = 8333; // Min for Zivid One
        if(modelName.substr(0, 14) == "Zivid One Plus")
            exposureTime = 6500; // Min for Zivid One Plus

        return exposureTime;
```

but, I'll take your change.